DSharpPlus.Commands/Processors/BaseCommandProcessor/BaseCommandProcessor.cs

```diff
@@ -223,6 +223,18 @@ await this.extension.commandErrored.InvokeAsync(converterContext.Extension, new
                 parsedArguments.Add(converterContext.Parameter, optional.RawValue);
             }
 
+            if (parsedArguments.Count == 0)
+            {
+                foreach (CommandParameter parameter in converterContext.Command.Parameters)
+                {
+                    parsedArguments.Add
+                    (
+                        parameter,
+                        parameter.DefaultValue.HasValue ? parameter.DefaultValue.Value : null

```

Comment:
This should check that all parameters are optional instead of passing `null` for required parameters.